app/src/main/java/nerd/tuxmobil/fahrplan/congress/system/OnLocaleChangedReceiver.java

```diff
@@ -0,0 +1,34 @@
+package nerd.tuxmobil.fahrplan.congress.system;
+
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+
+import org.ligi.tracedroid.logging.Log;
+
+import nerd.tuxmobil.fahrplan.congress.notifications.NotificationHelper;
+
+public final class OnLocaleChangedReceiver extends BroadcastReceiver {
```

Comment:
Please briefly explain the purpose of this class in [KDoc](https://kotlinlang.org/docs/reference/kotlin-doc.html) (once converted to Kotlin).

Reply:
Resolved in 3fa3217191e75ce614412947b95cb09160794cd9.